src/ModelButton/ModelButton.test.tsx

```diff
@@ -49,6 +50,50 @@ describe("ModelButton", () => {
     expect(svgElement).toHaveAttribute("stroke-width", "2");
   });
 
+  // test when status different than 'none' icon is rendered
+  test("does show status icon", () => {
+    render(<ModelButton status="error" />);
```

Comment:
Test below already doing the same. You can test no icon renderes if the status is `non`